ambry-api/src/main/java/com.github.ambry/config/HelixPropertyStoreConfig.java

```diff
@@ -51,16 +51,43 @@
   @Default("/ambry/defaultCluster/helixPropertyStore")
   public final String rootPath;
 
+  /**
+   * The time interval in second between two consecutive account pulling for the background account updater of
+   * {@code HelixAccountService}. Setting to 0 to disable it.
+   */
+  @Config(HELIX_PROPERTY_STORE_PREFIX + "account.service.polling.interval.ms")
+  @Default("60 * 60 * 1000")
+  // @todo This config by its nature should not appear in HelixPropertyStoreConfig. An ultimate fix would require
```

Comment:
why not do this now?

Reply:
Discussed this offline, would prefer to split it into two configs as they are not logically linked. To minimize the scope of this PR, will leave it as is, and create another PR just to split the config.

Reply:
Could you create an issue for this so that we can follow up?

Reply:
Also is 1 hr the intended value? Is that too long?

Reply:
Created #756

Reply:
One hour should be fine, consider the infrequency for user to update accounts. Also we have the listener to listen to the change, so most likely the updates will not be caught by this regular polling thread.

Reply:
I do have a metric that records if the update is captured by this thread, so can be used to adjust the default interval.